gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcombobox_ttt2.lua

```diff
@@ -56,10 +58,37 @@ function PANEL:Clear()
 	self.titleIndices = {}
 	self.valueIndices = {}
 	self.selected = nil
+	self.isValueTypeString = nil
 
 	self:CloseMenu()
 end
 
+---
+-- Stores the type of value that is currently used
+-- @param any value the value to store the type of
+-- @realm client
+function PANEL:StoreValueType(value)
+	-- Save value type of first entry for conversion
+	if self.isValueTypeString == nil then
```

Comment:
I don't think such a function should only allow storing the type once.

Reply:
But in the combobox case its the only thing that makes sense.
The first value that is stored determines the type until the combobox is cleared again.

A combobox should only support one type anyways.

Reply:
I don't agree here, because i think this should be able to handle every combination of types in different entries but i get your reasoning and why this is a fix. So im fine for now